ast/compile.go

```diff
@@ -536,6 +538,38 @@ func (c *Compiler) GetRulesWithPrefix(ref Ref) (rules []*Rule) {
 	return rules
 }
 
+func (c *Compiler) GetPackageAnnotations(pkg *Package) *Annotations {
+	as := c.annotationSet
+
+	result := make([]*Annotations, 0, 2)
+
+	result = append(result, as.getSubpackagesScope(pkg.Path)...)
+
+	if x := as.getPackageScope(pkg); x != nil {
+		result = append(result, x)
+	}
+
+	return mergeAnnotationsList(result)
+}
+
+func (c *Compiler) GetRuleAnnotations(rule *Rule) *Annotations {
+	as := c.annotationSet
+
+	result := make([]*Annotations, 0, 3)
+
+	if a := c.GetPackageAnnotations(rule.Module.Package); a != nil {
+		result = append(result, a)
+	}
```

Comment:
I've been wondering if it made sense to have a helper that would return a non-merged list of annotations so we could do
```go
result := []*Annotations{}
result = append(result, c.getPackageAnnotations(rule.Module.Package)...)
result = append(result, as.getDocumentScope(rule.Path())...)
result = append(result, as.getRuleScope(rule)...)
```

🤔 I doubt it's worth it. Let's keep it as-is.